src/core/ActivityInfoTree.h

```diff
@@ -46,6 +47,8 @@ class ActivityInfoTree : public QObject
 						const QDir &menuDir, const QString &menuFile);
 	void sortByDifficulty();
 	void sortByName();
+    QVariantList allCharacters();
+    QVariantList keyboardcharacters;
```

Comment:
Need to be prefixed with m_ to know it's a class attribute. Need to use camelCase also for the name.